murder-mystery.py

```diff
@@ -0,0 +1,29 @@
+name1 = input('enter a name')
+adj1 = input('enter an adjective')
+animal1 = input('enter plural animals')
+verb1 = input('enter past tense verb')
+con1 = input('enter a conjunction')
```

Comment:
why did you want to use a conjunction here?

Reply:
BUT no matter how hard